src/test/java/kg/jarkyn/server/Request/RequestParserTest.java

```diff
@@ -0,0 +1,31 @@
+package kg.jarkyn.server.Request;
+
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class RequestParserTest {
+
+    private String requestString = "GET /index.html HTTP/1.1\n" +
```

Comment:
Since the other headers are not used or exercised in the tests, this example request can have less content.